From 7563e0bbb54fbac54e77f3a62d9761a70f3a559c Mon Sep 17 00:00:00 2001
From: Colin Guthrie <colin@mageia.org>
Date: Sat, 01 Oct 2011 11:03:44 +0000
Subject: libpulse: Always return a three part version number in API calls.

For both the headers and the library we should provide clean, three part
strings as this has been what we've previously done in the past
and some external systems apparently rely on this format. While it's not
something we've officially commented on before, there is no real advantage
to us to change it so let's not try to tidy things up too much
considering some third party apps (e.g. Skype) seem to dislike a two
part version string.
---
(limited to 'src/pulse/version.h.in')

diff --git a/src/pulse/version.h.in b/src/pulse/version.h.in
index 7e00c5e..1be4c75 100644
--- a/src/pulse/version.h.in
+++ b/src/pulse/version.h.in
@@ -35,7 +35,7 @@ PA_C_DECL_BEGIN
 /** Return the version of the header files. Keep in mind that this is
 a macro and not a function, so it is impossible to get the pointer of
 it. */
-#define pa_get_headers_version() ("@PACKAGE_VERSION@")
+#define pa_get_headers_version() ("@PA_MAJOR@.@PA_MINOR@.0")
 
 /** Return the version of the library the current application is
  * linked to. */
--
cgit v0.9.0.2-2-gbebe
